src/WordPress.php

```diff
@@ -763,6 +763,9 @@ private function wp_download_and_config( $assoc_args ) {
 			if ( ! \EE::exec( $wp_config_create_command ) ) {
 				throw new \Exception( sprintf( 'Couldn\'t connect to %s:%s or there was issue in `wp config create`. Please check logs.', $this->site_data['db_host'], $this->site_data['db_port'] ) );
 			}
+			$old_wp_config_path = $this->site_data['site_fs_path'] . '/app/htdocs/wp-config.php';
```

Comment:
Just rename this to `$default_wp_config_path` since it's not old path it just default path,